scripts/vagrant/Vagrantfile

```diff
@@ -3,8 +3,13 @@
 
 Vagrant.configure("2") do |config|
 
-  config.vm.synced_folder "../..", "/go/src/github.com/ligato/networkservicemesh"
+  config.vm.synced_folder "../..", "/go/src/github.com/ligato/networkservicemesh", type: "sshfs"
+  config.vm.synced_folder ".", "/vagrant", type: "sshfs"
```

Comment:
`config.vm.allowed_synced_folder_types = [:sshfs]`
or you can do it in the `libvirt` section. See below.

Reply:
Oh even better I guess you should be able to do something like:
`config.vm.allowed_synced_folder_types = [:virtualbox, :sshfs]`

So when you use libvirt it will fail on the fu=irst available sync type and will try the sshfs one.

Reply:
Thanks, for your review @nickolaev, I'll test your proposals and update my PR.